core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -605,12 +640,14 @@ import scala.util.control.NonFatal
       val tps = partitions.asScala.toSet
       commitRefreshing.assignedPositions(tps, consumer, positionTimeout)
       listener.onAssign(tps)
+      rebalanceInProgress = false
```

Comment:
Even without any partitions assigned, this will be called with an empty collection, right? That will make it sure that actor does not get stuck in rebalance-in-progress state.

Reply:
Yes.